src/navigation/components/Header/Header.tsx

```diff
@@ -114,8 +117,21 @@ export const Header: React.FC<HeaderProps> = ({data, logo}) => {
         setIsSidebarOpened(false);
     }, []);
 
+    useEffect(() => {
+        const handleScroll = () => {
+            if (window.scrollY > 0 && !withBorder) {
+                setWithHeaderBorder(true);
```

Comment:
why do we set withHeaderBorder when withBorder is false?

Reply:
Because `withBorder = false` needs only for `scrollY = 0`. If we have  `scrollY > 0` we have to show border all the time

Reply:
` If we have scrollY > 0 we have to show border all the time`
then should this condition be like `if(window.scrollY > 0)`?

Reply:
It's not necessary. If `withBorder = true` we don't have to  change anything.
`const [showBorder, setShowBorder] = useState(withBorder);`
`showBorder` is constantly `true`

Reply:
I think we can check `showBorder ` instead of `withBorder ` if you think so